apis/sesv2/v1alpha1/zz_email_template.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by ack-generate. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// EmailTemplateParameters defines the desired state of EmailTemplate
+type EmailTemplateParameters struct {
+	// Region is which region the EmailTemplate will be created.
+	// +kubebuilder:validation:Required
+	Region string `json:"region"`
+	// The content of the email template, composed of a subject line, an HTML part,
+	// and a text-only part.
+	// +kubebuilder:validation:Required
+	TemplateContent *EmailTemplateContent `json:"templateContent"`
+	// The name of the template you want to create.
+	// +kubebuilder:validation:Required
+	TemplateName                  *string `json:"templateName"`
```

Comment:
This should be taken from [external name annotation](https://github.com/crossplane/provider-aws/blob/master/CODE_GENERATION.md#external-name) I think.

Reply:
i have changed using external-name annotation referencing